services/queue/src/api.js

```diff
@@ -1406,6 +1406,7 @@ builder.declare({
     );
   }
 
+  await this.queueService.removePendingMessage(taskId, runId);
```

Comment:
Should this have always been here and was noticed in implementing this PR? Seems like a bug fix

Reply:
this is only because this method is long being deprecated, and I believe only tests are using this method to claim specific task by taskId. 
Plus tests never tested the exact number of pending tasks because of the extra complexity of the data storage in the past (base64 encoded json strings with different messages in a single azure table)

Reply:
Cool thanks for the info 👍🏻